parse_opcodes

```diff
@@ -36,7 +36,8 @@ arglut['zimm'] = (19,15)
 arglut['shamt'] = (25,20)
 arglut['shamtw'] = (24,20)
 arglut['imm5'] = (24,20)
-arglut['imm6'] = (25,20)
+arglut['imm6hi'] = (25,25)
+arglut['imm6lo'] = (24,20)
```

Comment:
One could argue that `hi` and `lo` are switched, depending on whether we look at it from the final immediate's perspective or the instruction's. If I understand correctly, we are currently aligned with how the `bimm12` is handled at the moment (which is also shuffled), so this should be fine. Can you confirm this?

Reply:
If I look at jimm20 then it looks like the split of imm6 is not needed anyway. With this shuffle function I can just rearrange the bits of imm6 like I please.